scss/4-elements/_typography.scss

```diff
@@ -1,59 +1,62 @@
 %cads-h1 {
   font-size: 2.5rem;
-  line-height: 1.2em; // Line-height is variable
+  line-height: 3.25rem;
   font-weight: $cads-font-weight__bold;
   color: $cads-font-colour__headings;
   margin: 0 0 $cads-spacing-7 0;
 
   @include cads-media-breakpoint-down(sm) {
     font-size: 2rem;
+    line-height: 2.5rem;
     margin: 0 0 $cads-spacing-5 0;
   }
 }
 
 %cads-h2 {
   font-size: 2rem;
-  line-height: 1.25em; // Line-height is variable
+  line-height: 2.75rem;
   font-weight: $cads-font-weight__bold;
   color: $cads-font-colour__headings;
   margin: 0 0 $cads-spacing-4 0;
 
   @include cads-media-breakpoint-down(sm) {
     font-size: 1.5rem;
+    line-height: 2rem;
     margin: 0 0 $cads-spacing-3 0;
```

Comment:
This could be removed as the margin should be the same as the md and lg breakpoints (`margin: 0 0 $cads-spacing-4 0`)